README.md

```diff
@@ -7,22 +7,18 @@ This repository contains a `Dockerfile` designed to support Garry's Mod Dedicate
 First, build the container image with:
 
 ```bash
-$ docker build -t quay.io/rbreslow/garrysmodds:slim .
+docker build -t quay.io/rbreslow/garrysmodds:slim .
 ```
 
 Then, run the server and expose ports:
 
 ```bash
-$ docker run -ti \
+docker run -ti \
     -p 27015:27015 \
     -p 27015:27015/udp \
-    -p 27020:27020/udp \
```

Comment:
These ports aren't necessary for running SRCDS for demonstration purposes.

See: https://developer.valvesoftware.com/wiki/Source_Dedicated_Server#Connectivity

Reply:
:+1: